schema/crdb/dbinit.sql

```diff
@@ -2371,6 +2380,8 @@ CREATE TABLE IF NOT EXISTS omicron.public.tuf_artifact (
     PRIMARY KEY (name, version, kind)
 );
 
+CREATE INDEX IF NOT EXISTS tuf_artifact_sha256 ON omicron.public.tuf_artifact (sha256);
```

Comment:
This has the effect of adding a UNIQUE constraint to the `sha256` column, which causes the bogus repos we generate during testing to fail to upload (because many of the bogus artifacts have the same sha256 checksum despite having different name/version/kind values). Should I instead be paginating on the composite primary key?